doc/docs/Python_Tutorials/Basics.md

```diff
@@ -1201,9 +1200,9 @@ from mayavi import mlab
 s = mlab.contour3d(eps_data, colormap="YlGnBu")
 mlab.show()
 ```
-<p align="center">
-  <img src="../images/prism_epsilon.png">
-</p>
+
+![](../images/prism_epsilon.png#center)
```

Comment:
With this change, the image reappears in https://meep.readthedocs.io/en/latest/Python_Tutorials/Basics/#visualizing-3d-structures but is *not* centered.